tests/test_unit_tests.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+
+@pytest.mark.parametrize(
+    'dynasty',
+    [
+        (0),
+        (1),
+        (99999),
+        (-100),
+        (-1),
+    ]
+)
+def test_dynasty_wei_delta_defaults(casper, dynasty):
```

Comment:
Not sure the purpose of this test

Reply:
Was a sanity check when I added the new var early in dev. Will remove